spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1010,7 +1010,7 @@ def main(args=None):
         return
 
     # logger
-    spot_ros.wrapper_logger = rcutils_logger.RcutilsLogger(name="spot_wrapper")
+    spot_ros.wrapper_logger = rcutils_logger.RcutilsLogger(name=f"{spot_ros.name}.spot_wrapper")
```

Comment:
does this look funny if we doing a single robot and didn't provide a robot name?

Reply:
Good point! Will fix.

Reply:
addressed in the new commit. Tested both cases.